lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -2616,7 +2712,7 @@ class ConvertTorchToLinalg
     patterns.add<ConvertAtenEmbeddingOp>(typeConverter, context);
     target.addIllegalOp<AtenOnesOp>();
     patterns.add<ConvertAtenOnesOp>(typeConverter, context);
-
+    patterns.add<ConvertAtenToDtypeOp>(typeConverter, context);
```

Comment:
I think the op should also be marked as illegal here

Reply:
Done.